### PR TITLE
A few small edits to the GitLab section

### PR DESCRIPTION
**Pull request by selhorn**
Editing to ensure full sentences and fix a missing word.

### CODE REVIEW

**Comment by selhorn** [comment]
@mjang I could not figure out how to assign to you. Can you take a look?

File: docs/conf/portland/2021/writing-day.rst [hunk 1]
```diff
@@ -49,29 +49,29 @@ Write the Docs Meetups
...
+`GitLab.com account <https://gitlab.com/users/sign_up>`_ to get started.
 
 For more information, see our `GitLab issue <https://gitlab.com/gitlab-org/technical-writing/-/issues/426>`_.
 
-Once you find the issue you want to work on:
+When you find an issue you want to work on:
 
-#. Add a comment mentioning `@gl-docsteam` to inform us that you’re working on
-   this for Writing Day (and tell us in person!).
+#. To inform the GitLab team that you're working on an issue,
```
Comment (mjang) [inline docs/conf/portland/2021/writing-day.rst]: To make things parallel

```suggestion
#. Inform the GitLab team that you're working on an issue.
```

File: docs/conf/portland/2021/writing-day.rst [hunk 2]
```diff
@@ -49,29 +49,29 @@ Write the Docs Meetups
...
+   add a comment that mentions `@gl-docsteam` (or tell us in person!).
```
Comment (mjang) [inline docs/conf/portland/2021/writing-day.rst]:
```suggestion
   Add a comment that mentions `@gl-docsteam` (or tell us in person!).
```

**Review by mjang — APPROVED** [review]
Comments discussed and approved.

**Pull request merged**